extension/src/resources/components/settings/chat-appearance.tsx

```diff
@@ -3,25 +3,35 @@ import { cn } from "@Shad/lib/utils";
 import type { TwitchUser } from "~types/types";
 
 import { t } from "~utils/i18nUtils";
+import { env } from "../../../config/env";
```

Comment:
```suggestion
import { env } from "@Config/env";
```

To keep everything using alias ^^